web/themes/custom/votegov/src/sass/components/hero.scss

```diff
@@ -20,9 +20,10 @@
 }
 
 .vote-hero__image {
+  @include u-order(2);
```

Comment:
If the hero image is always expected to be below on mobile then we need to update the paragraph template so the image comes second in the dom and we can remove this reordering altogether. Then we can make sure we are only setting the container to flex on tablet-lg breakpoint and above using mobile-first approach.